lib/membrane_rtc_engine/endpoints/webrtc/connection_allocator.ex

```diff
@@ -85,4 +85,11 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.ConnectionAllocator do
   to the track receiver that gets new allocation.
   """
   @callback request_allocation(pid(), number()) :: :ok
+
+  @doc """
+  A function used by VariantSelector to change it's negotiability status.
+
+  TrackReceiver is considered negotiable if it is both capable of decreasing it's bandwidth usage and, in principal, allowed to do so.
```

Comment:
```suggestion
  TrackReceiver is considered negotiable if it is both capable of decreasing its bandwidth usage and, in principal, allowed to do so.
```

Please check if TrackReceiver is linked correctly in the documentation